### PR TITLE
CloudWatch: Added AWS RoboMaker metrics and dimensions

### PR DESCRIPTION
**Pull request by ilyastoli**
What this PR does / why we need it:
Adding AWS RoboMaker metics according to: https://docs.aws.amazon.com/robomaker/latest/dg/monitoring-aws-robomaker-cloudwatch.html
Which issue(s) this PR fixes:
No issue

Special notes for your reviewer:
Based on AWS documentation only, I didn't tried to monitor AWS RoboMaker myself

### CODE REVIEW

File: pkg/tsdb/cloudwatch/metric_find_query.go [hunk 1]
```diff
@@ -98,6 +98,7 @@ var metricsMap = map[string][]string{
...
 	"AWS/Redshift":          {"CPUUtilization", "DatabaseConnections", "HealthStatus", "MaintenanceMode", "NetworkReceiveThroughput", "NetworkTransmitThroughput", "PercentageDiskSpaceUsed", "QueriesCompletedPerSecond", "QueryDuration", "QueryRuntimeBreakdown", "ReadIOPS", "ReadLatency", "ReadThroughput", "TotalTableCount", "WLMQueriesCompletedPerSecond", "WLMQueryDuration", "WLMQueueLength", "WriteIOPS", "WriteLatency", "WriteThroughput"},
+	"AWS/Robomaker":         {"RealTimeFactor", "vCPU", "Memory", "SimulationUnit", "ResourceCount"},
```
Comment (marefr) [inline pkg/tsdb/cloudwatch/metric_find_query.go:101]: I only see `ResourceCount` metric being mentioned in related to `AWS/Usage` and not `AWS/Robomake` in the documentation. Can you explain why you've added it here?

Comment (ilyastoli) [reply pkg/tsdb/cloudwatch/metric_find_query.go:101]: Yes, you are right, my mistake, i'll fix that

**Review by marefr — APPROVED** [review]
LGTM

**Comment by marefr** [comment]
Thank you for contributing to Grafana!

**Pull request merged**